scalameta/inputs/shared/src/main/scala/scala/meta/inputs/Input.scala

```diff
@@ -19,9 +19,9 @@ object Input {
     override def toString = "Input.None"
   }
 
-  @leaf class String(s: scala.Predef.String) extends Input {
-    lazy val chars = s.toArray
-    override def toString = s"""Input.String("$s")"""
+  @leaf class String(value: scala.Predef.String) extends Input {
```

Comment:
While you're at it, can you add the entries from here https://github.com/scalameta/scalameta/pull/907#issuecomment-306212404 and maybe add some more changes in Trees that I missed?